### PR TITLE
Data schema clarification

### PR DESCRIPTION
**Pull request by lu-zero**
Minimal change to address #1817 + some whitespace cleanups.

***
<a href="https://pr-preview.s3.amazonaws.com/luminem/wot-thing-description/pull/1842.html" title="Last updated on Jul 12, 2023, 3:17 PM UTC (5f5125c)">Preview</a> | <a href="https://pr-preview.s3.amazonaws.com/w3c/wot-thing-description/1842/a09239f...luminem:5f5125c.html" title="Last updated on Jul 12, 2023, 3:17 PM UTC (5f5125c)">Diff</a>

### CODE REVIEW

**Review by egekorkan — APPROVED** [review]

**Review by relu91 — APPROVED** [review]

File: index.template.html [hunk 1]
```diff
@@ -1223,29 +1223,31 @@ <h2>Core Vocabulary Definitions</h2>
...
-    
+
       <section id="sec-data-schema-vocabulary-definition">
       <h2>Data Schema Vocabulary Definitions</h2>
 	  <p>A data schema is an abstract notation for data contained in data formats.</p>
       <p>
-        The data schema vocabulary definition reflects a very common subset of the terms defined by JSON Schema [[?JSON-SCHEMA]]. It is noted
+        The data schema vocabulary definition reflects a very common subset of the terms defined by JSON Schema [[?JSON-SCHEMA]].
+        A JSON Schema [[?JSON-SCHEMA]] processor for <a href="https://json-schema.org/specification-links.html#draft-7">JSON Schema draft 7</a>
+        can consume a Data Schema. It is noted
```
Comment (danielpeintner) [inline index.template.html:1233]:
```suggestion
        can consume a data schema. It is noted
```

It seems to be lowercase everywhere else...

**Review by danielpeintner — CHANGES_REQUESTED** [review]

File: index.template.html (hunk 1 above)
Comment (lu-zero) [reply index.template.html]: Thank you :)

**Review by danielpeintner — APPROVED** [review]

**Comment by egekorkan** [comment]
In the call of 05.07, we agreed that this helps the implementors. It was a bug to not mention this clearly in the beginning. We should add this to the REC version with a group-wide resolution.

We should see if there are other such dependencies.

In 2.0, we should ideally have an implementation guidance section.

**Comment by egekorkan** [comment]
Merging as communicated at https://lists.w3.org/Archives/Member/member-wot-wg/2023Aug/0003.html

**Pull request merged**